From 349df43e7fb528b335f02c5e95b55aa93f792c7d Mon Sep 17 00:00:00 2001
From: Fangrui Song
Date: Sun, 18 Nov 2018 13:48:31 -0800
Subject: [PATCH 2/7] Fix "const const" warning

---
 fs_test/lwext4_server.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs_test/lwext4_server.c b/fs_test/lwext4_server.c
index 2c923ba..b65ef3d 100644
--- a/fs_test/lwext4_server.c
+++ b/fs_test/lwext4_server.c
@@ -626,7 +626,7 @@ static int file_read(const char *p)
 	return rc;
 }
 
-static int file_write(const const char *p)
+static int file_write(const char *p)
 {
 	int fid = MAX_FILES;
 	int d;
-- 
2.19.1

