From c6bb451ea04b812485d5b31a69eb433498f8eabe Mon Sep 17 00:00:00 2001
From: Fangrui Song
Date: Sat, 17 Nov 2018 15:03:32 -0800
Subject: [PATCH 3/7] Don't copy include

---
 CMakeLists.txt | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 71eb24d..eec0993 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2,6 +2,7 @@ project(lwext4 C)
 cmake_minimum_required(VERSION 3.4)
 
 
+include_directories(include)
 include_directories(${PROJECT_BINARY_DIR}/include)
 include_directories(blockdev/filedev)
 include_directories(blockdev/filedev_win)
@@ -84,12 +85,6 @@ else()
     set_target_properties(lwext4 PROPERTIES COMPILE_FLAGS "-Wall -Wextra -pedantic")
 endif()
 
-#Config file generation
-file(
-    COPY include
-    DESTINATION .
-)
-
 #DISTRIBUTION
 set(CPACK_PACKAGE_VERSION_MAJOR "${VERSION_MAJOR}")
 set(CPACK_PACKAGE_VERSION_MINOR "${VERSION_MINOR}")
-- 
2.19.1

